frontend/src/scenes/plugins/edit/PluginDrawer.tsx

```diff
@@ -95,7 +93,7 @@ export function PluginDrawer(): JSX.Element {
                     <>
                         <div style={{ display: 'flex' }}>
                             <div style={{ flexGrow: 1 }}>
-                                {canDelete && (
+                                {editingPlugin?.organization_id === user?.organization?.id && (
```

Comment:
Might be wise to show this only for `INSTALL` and greater?

Reply:
Alright, it's very much an edge case like the other comment, but added access level checks to both places.